github/README.md

```diff
@@ -82,14 +89,6 @@ Create an issue.
 
 ![GitHub create_issue command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/create_issue.png)
 
-### [`github_find_pr`](https://github.com/nimbella/command-sets/blob/master/github/packages/github/github_find_pr.js)

```

Comment:
why is this removed?

Reply:
It had similar functionality as [github_view_pr.js](https://github.com/nimbella/command-sets/blob/master/github/packages/github/github_view_pr.js) and the actual implementation file (github_find_pr.js) was removed long back, only the document was not updated.